src/hooks/useServerApi.ts

```
@@ -162,41 +171,42 @@
+    additionalHeaders?: Record<string, string>
   ): Promise<T> => {
     return callApi<T>({
       method: 'POST',
       url,
       data: payload,
       authRequired: true,
-      impersonationData,
+      additionalHeaders,
     });
   };
   const getApi = async <T, P extends Record<string, string> = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)